### DISCUSSION TITLE
Repo is not indexed by Google

### DISCUSSION THREAD

**Discussion by gpisano97** [community/community#38541] [Other Feature Feedback, Questions, & Ideas]
Hello, i've published my new project on Github with MIT License. This is my first public project and i'm not expert with this. I noticed that my repo isn't reached by Google. So i make a test with "site:" command and nothing found. How can I achieve Google indexing ? Thank's for the support.

**Comment by SilasRodrigues19** [comment]
@gpisano97 To improve the indexing of the repository, you can create a README detailing the project well and in the description put a summary of the project with some topics as well.

I recommend you [this template](https://github.com/othneildrew/Best-README-Template) to create the README and if you want you can look at my repositories to get an idea of ​​what you can put in your readme and in the repository topics

**Reply by gpisano97** [reply]
Already done, my repo url is : https://github.com/gpisano97/Micron

**Reply by SilasRodrigues19** [reply]
Regarding the website, you can create some meta tags that help with this:

`<meta name="author" content="Your name">`
`<meta name="description" content="Site description">`
`<meta name="copyright" content="Your name 2022">`
`<meta name="keywords" content="keyword1, keyword2, keyword3, ...">`

**Reply by SilasRodrigues19** [reply]
Add topics to your repo, it's missing
